.github/workflows/renovate.yml

```
@@ -64,7 +64,11 @@ jobs:
 
       - uses: renovatebot/github-action@v40.1.12
         with:
-          configurationFile: .github/renovate.json
+          # Renovate recommends _not_ to name this one of the repository configuration file names:
+          # > It is recommended to not name it one of the repository configuration filenames listed in the Renovate Docs for Configuration Options.
+          # >
+          # > — https://github.com/renovatebot/github-action?tab=readme-ov-file#configurationfile
```

I found this comment very hard to understand.

```suggestion
          # Renovate recommends _not_ to use any of
          # [these names](https://docs.renovatebot.com/configuration-options/).
```

---

Thanks for the suggestion!